bundles/org.openhab.ui/web/src/pages/settings/pages/pages-list.vue

```diff
@@ -2,6 +2,8 @@
   <f7-page @page:afterin="onPageAfterIn" @page:afterout="onPageAfterOut">
     <f7-navbar title="Pages" back-link="Settings" back-link-url="/settings/" back-link-force>
       <f7-nav-right>
+        <f7-link v-if="$store.state.developerDock" icon-f7="question_circle_fill" @click="$f7.emit('toggleDeveloperDock')" />
+        <f7-link v-else icon-f7="question_circle" @click="$f7.emit('selectDeveloperDock',{'dock':'help','helpTab':'current'})" />
```

Comment:
How about when the developer dock can't be displayed because the screen is too narrow?
This new button/link will still be displayed but appear to (and in fact will) do nothing I think, which is not good.
Maybe best remove them when the current screen is not able to display the dock - or keep them but just link them to relevant documentation instead.

Reply:
Oh yeah, forgot that point from the previous discussion.  If it's all the same to you, I'm inclined to just remove it so it doesn't clutter the navbar on narrower screens.

Reply:
I do think it's best to either remove it, or perhaps change it to a regular link to the documentation when the developer dock wouldn't fit.

By the way, I think when those icons make their way in the navbar, we should replace these buttons in the empty state placeholders:

![image](https://github.com/openhab/openhab-webui/assets/2004147/401b2f30-1c31-4fd7-90d4-d1e9e340f1b9)

You have a message saying "add your first thing manually with the button below" but the actual button below opens the documentation. It was an oversight which should be corrected.

Reply:
> By the way, I think when those icons make their way in the navbar, we should replace these buttons in the empty state placeholders:

I hadn't thought of that, that's a good point.

> You have a message saying "add your first thing manually with the button below" but the actual button below opens the documentation. It was an oversight which should be corrected.

Do you mean this?

![image](https://github.com/openhab/openhab-webui/assets/77952587/c6abf959-0bd5-4237-8176-da72b624604e)

The button is  link to the docs, but the "click here" refers to the list item itself.  Maybe that's not clear enough...

Otherwise, I'm not sure which button you're referring to.

Reply:
> By the way, I think when those icons make their way in the navbar, we should replace these buttons in the empty state placeholders:

Since we decided to just remove the button on narrow screens, I only removed the documentation buttons after the placeholders when the navbar help buttons are visible.  So, on narrow screens instead of the navbar button you will still see the documentation buttons.

> > You have a message saying "add your first thing manually with the button below" but the actual button below opens the documentation. It was an oversight which should be corrected.
> 
> Do you mean this?

I'm still not 100% sure which message you are referring to, but I've added arrow indicators to all the `click here` list items in the quickstart section which I hope will make it more clear that those specific items are clickable.

![image](https://github.com/openhab/openhab-webui/assets/77952587/7b0e1e19-2808-4019-abd5-5c0544e86eb6)